lib/src/screens/emailVerification.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+import 'package:flutter_platform_widgets/flutter_platform_widgets.dart';
+import 'package:oneplace_illinois/src/misc/colors.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:oneplace_illinois/src/services/firebaseAuth.dart';
+
+class EmailVerification extends StatefulWidget {
+  const EmailVerification({Key? key}) : super(key: key);
+
+  @override
+  _EmailVerificationState createState() => _EmailVerificationState();
+}
+
+class _EmailVerificationState extends State<EmailVerification> {
+  final FirebaseAuthService _authService = FirebaseAuthService();
+
+  @override
+  Widget build(BuildContext context) {
+    return PlatformScaffold(
+        appBar: PlatformAppBar(
+          backgroundColor: AppColors.secondaryUofIDark,
+          title: Text(
+            "Verify Email",
+            style: TextStyle(color: Colors.white),
+          ),
+        ),
+        iosContentPadding: true,
+        body: Row(

```

Comment:
Why is there a `Row` here if the only child is the `Column`?  If you wanted to center the column try doing this

```suggestion
        body: Center(
              child: Column(
```